src/test_helpers/unit.cljs

```diff
@@ -6,14 +6,15 @@
   prefer to use it for more general purpose concepts, such as the re-frame event
   layer."
   (:require-macros test-helpers.unit)
+  ;; We must require test-helpers.matchers namespace to register the custom cljs.test directive
```

Comment:
This comment moved because there is some discrepancy between local `make lint-fix` and PR check. And local lint tries to change it in a way that is not accepted in PR :-|